### PR TITLE
Cleanup custom properties

### PR DESCRIPTION
**Pull request by javalkar-varun**
### Summary
There are multiple places where custom properties are not unique. They should be separate and themed within Layout.scss within the :root clause: https://github.com/cerner/terra-consumer/blob/master/packages/terra-consumer-layout/src/Layout.scss#L3

Cleanup variable names, adding new url for S3 bucket and removing redundant variables.

Naming convention followed to create custom property:
--terra-consumer-classname-propertyname

Link to the previous PR - https://github.com/cerner/terra-consumer/pull/52

Thanks for contributing to Terra. 
@cerner/terra

### CODE REVIEW

File: packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss [hunk 1]
```diff
@@ -107,8 +107,8 @@
 
     &:hover,
     &:focus {
-      background: var(--terra-consumer-profile-hover-background, rgba(255, 255, 255, 0.4));
-      color: var(--terra-consumer-profile-link-color, #3d3d3d);
+      background: var(--terra-consumer-popup-button-hover-hover-background, rgba(255, 255, 255, 0.4));
```
Comment (amf9t2) [inline packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss]: np: do we need two 'hover' here ?

File: packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss [hunk 2]
```diff
@@ -17,7 +17,7 @@
   }
 
   .img {
-    color: var(--terra-consumer-logo-text-color, #3d3d3d);
+    color: var(--terra-consumer-logo-img-color, #3d3d3d);
```
Comment (amf9t2) [inline packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss]: this color is applied on alternate text. If the src-url is valid then logo image will be rendered or else alt text will be rendered in this color.

File: packages/terra-consumer-nav/src/components/nav-items/NavItem.scss [hunk 3]
```diff
@@ -20,8 +20,7 @@
     text-decoration: none;
 
     &:hover {
-      background: var(--terra-consumer-link-hover-background, rgba(255, 255, 255, 0.4));
-      color: var(--terra-consumer-nav-link-color);
```
Comment (amf9t2) [inline packages/terra-consumer-nav/src/components/nav-items/NavItem.scss]: out of curiosity we don't want 'color' here ?

File: packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss (hunk 2 above)
Comment (amf9t2) [reply packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss]: we can use - `--terra-consumer-nav-text-color` here. Also mentioned in [jira description](https://jira2.cerner.com/browse/PORTALDEV-36272)

File: packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss (hunk 1 above)
Comment (javalkar-varun) [reply packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss]: no, only one. Changed it.

File: packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss (hunk 2 above)
Comment (javalkar-varun) [reply packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss]: -logo-text-color and -logo-img-color are different so I'm reverting it back to as it was earlier.

File: packages/terra-consumer-nav/src/components/nav-items/NavItem.scss (hunk 3 above)
Comment (javalkar-varun) [reply packages/terra-consumer-nav/src/components/nav-items/NavItem.scss]: yes, want color. Brought it back.

**Comment by KevinJJackson** [comment]
master:
image

this pr:
image

File: packages/terra-consumer-layout/src/Layout.scss [hunk 4]
```diff
@@ -2,14 +2,13 @@
...
   // Body
-  --terra-consumer-body-background-image: url('../background.jpg');
+  --terra-consumer-body-background-image: url('https://s3.amazonaws.com/healthelastic-prod/terra-consumer-background-images.jpg');
   --terra-consumer-body-background-color: #c7d4ea;
 
   // Nav
   --terra-consumer-nav-background-color: rgba(255, 255, 255, 0.25);
   --terra-consumer-nav-width: 320px;
   --terra-consumer-nav-link-color: #3d3d3d;
-  --terra-consumer-nav-active-link-background: var(--terra-consumer-body-background-image);
```
Comment (KevinJJackson) [inline packages/terra-consumer-layout/src/Layout.scss]: don't remove this line

**Comment by alex-bezek** [comment]
spoke with @mhemesath . He wants to not do this until after we release and can go back to re-factor things. Right now it will cause re-work could cause bugs

File: packages/terra-consumer-layout/src/Layout.scss (hunk 4 above)
Comment (javalkar-varun) [reply packages/terra-consumer-layout/src/Layout.scss]: done

**Comment by javalkar-varun** [comment]
Closing this issue as I see most of the cleanup for the custom properties in root clause is already handled - https://github.com/cerner/terra-consumer/blob/master/packages/terra-consumer-layout/src/Layout.scss 
Cc: @mhemesath @alex-bezek

**Pull request closed without merging**